### PR TITLE
MH-13194 wrong description for endpoint

### PR DESCRIPTION
**Pull request by tyfranck**
Corrected 'mediapackage' description for /ingest/{wdID} REST endpoint

### CODE REVIEW

File: modules/matterhorn-ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java [hunk 1]
```diff
@@ -1044,7 +1044,7 @@ public Response ingest(MultivaluedMap<String, String> formData) {
   @POST
   @Produces(MediaType.TEXT_HTML)
   @Path("ingest/{wdID}")
-  @RestQuery(name = "ingest", description = "Ingest the completed media package into the system, retrieving all URL-referenced files, and starting a specified workflow", pathParameters = { @RestParameter(description = "Workflow definition id", isRequired = true, name = "wdID", type = RestParameter.Type.STRING) }, restParameters = { @RestParameter(description = "The ID of the given media package", isRequired = true, name = "mediaPackage", type = RestParameter.Type.TEXT) }, reponses = {
+  @RestQuery(name = "ingest", description = "Ingest the completed media package into the system, retrieving all URL-referenced files, and starting a specified workflow", pathParameters = { @RestParameter(description = "Workflow definition id", isRequired = true, name = "wdID", type = RestParameter.Type.STRING) }, restParameters = { @RestParameter(description = "The given media package", isRequired = true, name = "mediaPackage", type = RestParameter.Type.TEXT) }, reponses = {
```
Comment (KatrinIhler) [inline modules/matterhorn-ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java]: Would you mind removing the 'given' to make it consistent with /ingest without a workflow id?

**Review by KatrinIhler — CHANGES_REQUESTED** [review]
Definitely a good idea to fix this. The merge commit confuses me though, why is that in here (especially since it doesn't seem to contribute anything)?

File: modules/matterhorn-ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java (hunk 1 above)
Comment (tyfranck) [reply modules/matterhorn-ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java]: Does your comment about this not contributing to anything related to this pull request or the previous one I closed?

Comment (KatrinIhler) [reply modules/matterhorn-ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java]: I'm specifically talking about the first of the two commits "Merge pull request #4 from opencast/r/4.x" - why is that here? What does it do?

**Comment by lkiesow** [comment]
The changes itself look just fine. @tyfranck, do you mind if we just squash-merge this? That will automatically drop the additional merge commits in here. I guess that would be okay with you as well, @KatrinIhler?

**Comment by KatrinIhler** [comment]
@lkiesow Sounds good to me, but Github tells me that squash & merge is not enabled for this repository, so how do we do this?

**Comment by tyfranck** [comment]
@lkiesow  please go ahead. I've found a way to avoid those extra commits. So next time will be a cleaner one... :-)

**Comment by lkiesow** [comment]
@KatrinIhler, found a way to do that.
Sometimes, being an admin helps ;-)

**Pull request merged**